pandas/tests/io/pytables/test_store.py

```diff
@@ -39,4989 +25,996 @@
-            tm.assert_frame_equal(store["df"], df)
+        assert type(df2.index.name) == str
+        assert type(df2.columns.name) == str
 
-            _maybe_remove(store, "df2")
-            store.append("df2", df[:10], dropna=False)
-            store.append("df2", df[10:], dropna=False)
-            tm.assert_frame_equal(store["df2"], df)
 
-    def test_store_dropna(self, setup_path):
-        df_with_missing = DataFrame(
-            {"col1": [0.0, np.nan, 2.0], "col2": [1.0, np.nan, np.nan]},
-            index=list("abc"),
-        )
-        df_without_missing = DataFrame(
-            {"col1": [0.0, 2.0], "col2": [1.0, np.nan]}, index=list("ac")
-        )
+def test_store_series_name(setup_path):
+    df = tm.makeDataFrame()
+    series = df["A"]
 
-        # # Test to make sure defaults are to not drop.
-        # # Corresponding to Issue 9382
-        with ensure_clean_path(setup_path) as path:
-            df_with_missing.to_hdf(path, "df", format="table")
-            reloaded = read_hdf(path, "df")
-            tm.assert_frame_equal(df_with_missing, reloaded)
+    with ensure_clean_store(setup_path) as store:
+        store["series"] = series
+        recons = store["series"]
+        tm.assert_series_equal(recons, series)
 
-        with ensure_clean_path(setup_path) as path:
-            df_with_missing.to_hdf(path, "df", format="table", dropna=False)
-            reloaded = read_hdf(path, "df")
-            tm.assert_frame_equal(df_with_missing, reloaded)
 
-        with ensure_clean_path(setup_path) as path:
-            df_with_missing.to_hdf(path, "df", format="table", dropna=True)
-            reloaded = read_hdf(path, "df")
-            tm.assert_frame_equal(df_without_missing, reloaded)
+@pytest.mark.filterwarnings("ignore:\\nduplicate:pandas.io.pytables.DuplicateWarning")
+def test_overwrite_node(setup_path):
 
-    def test_read_missing_key_close_store(self, setup_path):
-        # GH 25766
-        with ensure_clean_path(setup_path) as path:
-            df = DataFrame({"a": range(2), "b": range(2)})
-            df.to_hdf(path, "k1")
+    with ensure_clean_store(setup_path) as store:
+        store["a"] = tm.makeTimeDataFrame()
+        ts = tm.makeTimeSeries()
+        store["a"] = ts
 
-            with pytest.raises(KeyError, match="'No object named k2 in the file'"):
-                pd.read_hdf(path, "k2")
+        tm.assert_series_equal(store["a"], ts)
 
-            # smoke test to test that file is properly closed after
-            # read with KeyError before another write
-            df.to_hdf(path, "k2")
 
-    def test_read_missing_key_opened_store(self, setup_path):
-        # GH 28699
-        with ensure_clean_path(setup_path) as path:
-            df = DataFrame({"a": range(2), "b": range(2)})
-            df.to_hdf(path, "k1")
+@pytest.mark.filterwarnings(
+    "ignore:\\nthe :pandas.io.pytables.AttributeConflictWarning"
+)
+def test_coordinates(setup_path):
```

Comment:
I don't know what's going on with this test. I had intermittent failures on my computer, and then it failed in the Windows CI. I had a really hard time trying to debug this test because the failure was intermittent.

Reply:
Seems to have a common cause with #32225